cli/src/commands/get/comments.rs

```diff
@@ -216,6 +210,7 @@ fn download_comments(
                 &statistics,
                 options.include_predictions,
                 writer,
+                options,
```

Comment:
Passing `options.include_predictions` and also `options` duplicates the `include_predictions` argument and allows for inconsistent state / ussage where the argument `include_predictions` is different from what's passed in to the `options`.

I think you should either 
 - add ` TimeRange` argument and pass in `options.timerange` (as that's what you need) or 
 - pass in `options` and take `include_predictions` from the object

Given `CommentDownloadOptions` contains a bunch of other stuff you don't need:
```rust
#[derive(Default)]
struct CommentDownloadOptions {
    dataset_identifier: Option<DatasetIdentifier>,
    include_predictions: bool,
    reviewed_only: bool,
    timerange: CommentsIterTimerange,
    show_progress: bool,
}
```
I think you go with the former and just add a `timerange` argument

Reply:
Ah, if you only download review comments, you don't use the `timerage`, see above call to `get_reviewed_comments_in_bulk`